src/test/java/picard/sam/AbstractAlignmentMergerTest.java

```diff
@@ -1,15 +1,10 @@
 package picard.sam;
 
-import htsjdk.samtools.SAMFileHeader;
-import htsjdk.samtools.SAMFileWriter;
-import htsjdk.samtools.SAMFileWriterFactory;
-import htsjdk.samtools.SAMRecord;
-import htsjdk.samtools.SAMRecordIterator;
-import htsjdk.samtools.SAMRecordSetBuilder;
-import htsjdk.samtools.SamReader;
-import htsjdk.samtools.SamReaderFactory;
-import htsjdk.samtools.ValidationStringency;
+import htsjdk.samtools.*;
```

Comment:
there seems to be a preference for explicit imports rather than wildcard. you can change your intelliJ settings for that.

Reply:
this.